drivers/sensor/zephyr_thermistor/zephyr_ntc_thermistor_rt_table.c

```diff
@@ -127,6 +153,8 @@ int32_t ntc_get_temp_mc(const struct ntc_type *type, unsigned int ohm)
 	temp = ntc_fixp_linear_interpolate(type->comp[low].ohm, type->comp[low].temp_c * 1000,
 					   type->comp[high].ohm, type->comp[high].temp_c * 1000,
 					   ohm);
+	CLAMP(temp, min_temp, max_temp);
```

Comment:
You still need to assign it.
```suggestion
	temp = CLAMP(temp, min_temp, max_temp);
```